airbyte-webapp/src/components/ui/Markdown/remarkAdmonitionsPlugin.ts

```diff
@@ -11,7 +10,7 @@ import styles from "./admonitions.module.scss";
 const SUPPORTED_ADMONITION_NAMES: Readonly<string[]> = ["note", "tip", "info", "caution", "warning", "danger"];
 const SUPPORTED_NODE_TYPES: Readonly<string[]> = ["textDirective", "leafDirective", "containerDirective"];
 
-export const remarkAdmonitionsPlugin: Plugin<[], Root> = () => (tree) => {
+export const remarkAdmonitionsPlugin: Plugin<[]> = () => (tree) => {
```

Comment:
:information_source: Typings works very fine with the `Root` so no need to pull in `mdast` (which is also the deprecated version of remark) for that.